pillar/elastic_stack/beats/init.sls

```diff
@@ -0,0 +1,37 @@
+elastic_stack:
+  beats:
+    metricbeat:
+      config:
+        metricbeat.config.modules:
+          path: /etc/metricbeat/modules.d/*.yml
+          reload.enabled: true
+          reload.period: 30s
+        name: {{ grains['id'] }}
+        tags:
+          - {{ grains['id'] }}
+          - {{ grains.get('environment') }}
+          - {{ grains.get('roles') }}
+          - {{ grains.get('osfullname') }}
+        processors:
+          - add_cloud_metadata: ~
+          - add_host_metadata: ~
+        output.elasticsearch:
+          hosts:
+            - http://nearest-elasticsearch.service.operations.consul:9200
+          compression_level: 3
+      modules:
+        system:
+          - module: system
+            metricsets:
+              - cpu
+              - filesystem
+              - load
+              - memory
+              - network
+              - process
+              - process_summary
+              - uptime
+            enabled: 'true'
+            period: 1s
```

Comment:
Isn't this too low? Would for example `5s` or `10s` still give us a good picture?

Reply:
In order for it to be actually useful it's best to go as low as possible. If we can't handle the volume of writes at 1s granularity then we can try going coarser, but this is a good place to start.